packages/preview/clean-hda/0.1.0/README.md

```diff
@@ -0,0 +1,293 @@
+# Fork For Hochschule Darmstadt - University of Applied Sciences
+
+I recreated
+[mbredel/thesis-template](https://github.com/mbredel/thesis-template) in by
+forking the existing config of DHBW, which looked the most similar to the
+original latex implementation. 
+
+> [!NOTE]
+> This is an **unofficial** template for [Hochschule Darmstadt - University of Applied Sciences](www.h-da.de) for the department of Computer Sciences.
+> Contributions and takeover by h_da affiliated are welcome. 
+ 
+# How to include
+
+1. Add this as a git submodule
```

Comment:
I understand why this is included in the readme, but since you're submitting this package to typst universe, the standard way of using this package through the `@preview` namespace should be shown here. And using a git submodule could be advertised as an alternative option.